modules/gapi/samples/pipeline_modeling_tool.cpp

```diff
@@ -371,7 +380,10 @@ int main(int argc, char* argv[]) {
                     builder.addDummy(call_params, read<DummyParams>(node_fn));
                 } else if (node_type == "Infer") {
                     auto infer_params = read<InferParams>(node_fn);
-                    infer_params.config = config;
+                    RETHROW_WITH_MSG_IF_FAILED(
+                            utils::intersectMapWith(infer_params.config, gconfig),
+                            "Failed to combine global and local configs for Infer node: "
```

Comment:
Why it could fail? How user could deal with it?

Reply:
If user provides global and local configs that have intersection app fails with: `Met dublicated key` which is not descriptive. It would be great to see something like: `Failed to merge global & local configs for model <model.xml>`, but `utils::intersectMapWith` knows nothing about modes & configs it just merges `map`'s